### DISCUSSION TITLE
此贴无效【bug反馈】

### DISCUSSION THREAD

**Discussion by khiqwq** [share121/inter-knot#821] [其他]
主页可显示图片，当打开帖子后图片就不显示了，如图
![image](https://github.com/user-attachments/assets/f1d8856d-49af-41f2-86f1-c4de410b6191)
![image](https://github.com/user-attachments/assets/6113247f-8dc9-4581-a949-eef4c9147f68)

**Comment by khiqwq** [comment]
居然刷新一下就好了

**Comment by SDCOM-0415** [comment]
可以提交issues，大概率是GitHub服务器似了，瞬时压力比以往都大得多了
